src/main/kotlin/no/nav/familie/dokument/storage/StorageController.kt

```diff
@@ -21,56 +22,49 @@ import java.util.*
 
     /// TODO: "bucket"-path brukes ikke ennå. "familievedlegg" brukes alltid
     @PostMapping(path = ["{bucket}"],
                  consumes = [MediaType.MULTIPART_FORM_DATA_VALUE],
                  produces = [MediaType.APPLICATION_JSON_VALUE])
-    fun addAttachment(@PathVariable("bucket")bucket: String,
+    fun addAttachment(@PathVariable("bucket") bucket: String,
                       @RequestParam("file") multipartFile: MultipartFile): ResponseEntity<Map<String, String>> {
 
         if (multipartFile.isEmpty) {
-            log.info("Dokumentet som lastes opp er tomt - size: [{}] ", multipartFile.size)
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build()
+            throw IllegalArgumentException("Dokumentet som lastes opp er tomt - size: [${multipartFile.size}] ")
         }
 
         val bytes = multipartFile.bytes
-        val maxFileSizeInBytes = maxFileSizeInMb*1024*1024
+        val maxFileSizeInBytes = maxFileSizeInMb * 1024 * 1024
         log.debug("Dokument lastet opp med størrelse (bytes): " + bytes.size)
 
         if (bytes.size > maxFileSizeInBytes) {
-            throw IllegalArgumentException(HttpStatus.PAYLOAD_TOO_LARGE.toString())
+            throw IllegalArgumentException("Dokumentstørrelsen(${bytes.size} bytes) overstiger grensen(${maxFileSizeInMb} mb)")
         }
 
         val directory = contextHolder.hentFnr()
 
         val uuid = UUID.randomUUID().toString()
 
         val file = ByteArrayInputStream(bytes)
-        try {
-            storage.put(directory, uuid, file)
-        } catch (e: RuntimeException) {
-            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build()
-        }
-
-        return ResponseEntity.ok(mapOf("dokumentId" to uuid, "filnavn" to multipartFile.name))
+        storage.put(directory, uuid, file)
+        return ResponseEntity.status(HttpStatus.CREATED).body(mapOf("dokumentId" to uuid, "filnavn" to multipartFile.name))
     }
 
     /// TODO: "bucket"-path brukes ikke ennå. "familievedlegg" brukes alltid
     @GetMapping(path = ["{bucket}/{dokumentId}"], produces = [MediaType.APPLICATION_JSON_VALUE])
     fun getAttachment(@PathVariable("bucket") bucket: String,
-                      @PathVariable("dokumentId") dokumentId: String): ResponseEntity<Ressurs<ByteArray>> {
-        val directory = contextHolder.hentFnr()
+                      @PathVariable("dokumentId") dokumentId: String): ResponseEntity<ByteArray> {
         return try {
+            val directory = contextHolder.hentFnr()
             val data = storage[directory, dokumentId]
             log.debug("Loaded file with {}", data)
-            ResponseEntity.ok(Ressurs.Companion.success(data))
-        } catch (e: RuntimeException) {
-            secureLogger.error("Henting av vedlegg feilet", e)
-            ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(Ressurs.Companion.failure(e.message))
+            ResponseEntity.ok(data)
```

Comment:
This breaks the API, familie-ef-soknad-api will need to be changed if we remove the use of Ressurs here